libs/blocks/quiz-entry/quiz-entry.css

```diff
@@ -191,45 +284,59 @@ html[dir="rtl"] .quiz-option-icon {
   font-size: 14px;
   font-weight: 400;
   line-height: 21px;
-  margin: 0 0 0 17px;
+  margin: 0;
   text-align: start;
 }
 
-.quiz-option:hover .quiz-option-image {
-  filter: brightness(70%);
-}
-
-.quiz-option:hover .quiz-option-text {
-  display: block;
-} 
-
 .quiz-option:hover .quiz-option-icon {
   background-color: color-mix(in srgb, var(--quiz-icon-bg) 70%, var(--color-white));
 }
 
-.quiz-option:focus-visible {
-  background-color: #fafafaee;
-}
-
 .quiz-option:focus-visible .quiz-option-icon {
   background-color: color-mix(in srgb, var(--quiz-icon-bg) 80%, var(--color-white));
 }
 
 .quiz-option.selected {
-  border: 5px solid #FC00F2
+  border: 5px solid #FC00F2;
+  border-radius: 13px;
+}
+
+.quiz-option.selected::before {
+  background:  url("data:image/svg+xml,%3Csvg%20width%3D%2225%22%20height%3D%224%22%20viewBox%3D%220%200%2025%204%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Cpath%20d%3D%22M2%202H23%22%20stroke%3D%22white%22%20stroke-width%3D%224%22%20stroke-linecap%3D%22round%22%2F%3E%0A%3C%2Fsvg%3E") no-repeat;
+  background-size: 20px 4px;
+  content: " ";
+  height: 4px;
+  position: absolute;
+  right: 20px;
+  top: 20px;
+  width: 20px;
+  z-index: 1;
 }
 
 .quiz-option.selected .quiz-option-icon {
   background-color: color-mix(in srgb, var(--quiz-icon-bg) 70%, var(--color-white));
 }
 
-.quiz-option.selected .quiz-option-title {
-  color: var(--link-hover-color-dark);
+.quiz-option:hover .quiz-option-image, 
+.quiz-option.selected .quiz-option-image,
+.quiz-option:focus-visible .quiz-option-image {
+  filter: brightness(33%);
 }
 
-.quiz-option.selected .quiz-option-text {
-  color: var(--link-hover-color-dark);
-  font-weight: 700;
+.quiz-option:focus-visible::after {
+  content: " ";
+  border-radius: 0.5rem;
+  height: 100%;
+  outline: 4px solid #4B75FF;
+  outline-offset: 2px;
+  position: absolute;
+  width: 100%;
+  z-index: -1;
+}
+
+.quiz-option.selected:focus-visible::after {
+  border-radius: 8px;
+  outline-offset: 8px;
 }
 
 .quiz-option.disabled {
```

Comment:
for some reason, cursor: not-allowed is not working

Reply:
This issue is not with the css, but with the disabled state of the cards themselves. This seems to have been introduced with @fullcolorcoder's carousel pr, as you can see it happening even on the before URL. We should probably log a bug for this, or Brad can address it in hit carousel css refinement ticket.